examples/predict_video_on_local.py

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+
+import argparse
+import logging
+
+from geti_sdk.demos import predict_video_on_local
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("video_path", type=str)
+    parser.add_argument("deployment_path", type=str)
+    parser.add_argument("--device", choices=["CPU", "GPU"], default="CPU")
+    parser.add_argument("--drop_audio", action="store_true")
+    parser.add_argument("--log_level", choices=["warning", "info"], default="warning")
+
+    args = parser.parse_args()
+
+    level_config = {"warning": logging.WARNING, "info": logging.INFO}
+    log_level = level_config[args.log_level.lower()]
+    logging.basicConfig(level=log_level)
+
+    # --------------------------------------------------
+    # Configuration section
```

Comment:
This is not really in line with the configuration in the other examples. Currently, there is no configuration that the user has to do, because it's all specified in the command line args.

I suggest to make it so that it is possible to run the script against a live Geti instance as well, for which configuration (hostname, credentials, project name) would be required.

From my perspective, there are two use cases for the script:
1. The user passes a deployment path, and predictions are generated from there (no server configuration needed)
2. The user passes a Geti host, credentials, and a project name. A Deployment is created for the target project on the fly

The 2nd option requires additional configuration. Please have a look at the other examples to check how it's handled there, it's fine if users have to make minor edits in the scripts. All examples were set up like this.